releasenotes/notes/collect_extra_service_names-453190f684d62070.yaml

```diff
@@ -0,0 +1,7 @@
+---
+features:
+  - |
+    tracer: This introduces collection of fake service names.
```

Comment:
The word "fake" here is confusing for me. Is there any context you could add to this release note to clarify its meaning?

Reply:
This is related to the document "Service Names - Handling Inferred (Fake) Services".
If this is not appropriate for the release note, should I replace it with "Handling Inferred" ?